src/lib/translators/mongo.ts

```diff
@@ -527,6 +527,91 @@ function transformSplit(step: Readonly<S.SplitStep>): MongoStep {
   ];
 }
 
+/** transform a 'statistics' step into corresponding mongo steps */
+function transformStatistics(step: Readonly<S.StatisticsStep>): MongoStep {
+  // Some useful formula:
+  const quintilesIndexFormula: any = {
+    maxIndex: { $subtract: ['$count', 1] },
+    lastQuintileIndex: { $multiply: [{ $divide: ['$count', 10] }, 9] },
+    lastQuartileIndex: { $multiply: [{ $divide: ['$count', 4] }, 3] },
+    medianIndex: { $divide: ['$count', 2] },
+    firstQuartileIndex: { $divide: ['$count', 4] },
+    firstQuintileIndex: { $divide: ['$count', 10] },
+    minIndex: 0,
+  };
+  const varianceFormula: any = { $subtract: ['$average_sum_square', { $pow: ['$average', 2] }] };
+  const otherStatisticsFormula: any = {
+    count: 1,
+    average: 1,
+    variance: varianceFormula,
```

Comment:
what about a simple combination of `$pow` and [`$stdDevPop`](https://docs.mongodb.com/manual/reference/operator/aggregation/stdDevPop/) ? :)